internal/handlers/pg/msg_findandmodify.go

```diff
@@ -56,28 +58,49 @@ func (h *Handler) MsgFindAndModify(ctx context.Context, msg *wire.OpMsg) (*wire.
 		return nil, err
 	}
 
-	fetchedDocs, err := h.fetch(ctx, params.sqlParam)
-	if err != nil {
-		return nil, err
-	}
+	// TODO: SortDocuments requires everything :(
```

Comment:
Can you please add more context here? Is it about the fact that we need to load everything to perform in-memory sort?